src/smc-util-node/misc_node.coffee

```diff
@@ -847,10 +847,10 @@ exports.loadSync = (filename) ->
     JSON.parse(fs.readFileSync(filename).toString())
 
 
-# WA state sales tax rates, as of August 2015.
-# Generated scripts/sales_tax.py
+# WA state sales tax rates, as of April 20 2016.
+# Generated via http://dor.wa.gov/Content/FindTaxesAndRates/RetailSalesTax/DestinationBased/ClientInterface.aspx
```

Comment:
1. Yes. Patching with the script.
1. For whatever reason, these rates are different from another search tool that the DOR provides. This one returns lower values so I used this one.